meta/system.py

```diff
@@ -166,6 +166,15 @@ def set_kaldi_scorer(self, corpus, mapping):
         }
         self.scorer_hyp_arg[corpus] = "ctm"
 
+    def set_stm_files(self, corpus, **kwargs):
```

Comment:
From a set function I expect to have a parameter for the value that I want to set. A better name for what this function does is `create_stm_from_corpus`.